### ISSUE TITLE
Incorrect TODO parsing with multiple todo sets in one file

### ISSUE THREAD

**Issue by transitracer** [xiaoxinghu/orgajs#26]
Often it may make sense to have multiple sets of todo keywords in a file, but orga handles this incorrectly, with only the last parsed set of TODO keywords taking affect

Example from the org mode docs ( https://orgmode.org/manual/Per_002dfile-keywords.html )

```
#+TODO: TODO | DONE
#+TODO: REPORT BUG KNOWNCAUSE | FIXED
#+TODO: | CANCELED
```

Given how orga's updateTODOs function works, the previous TODOs are simply replaced with the newer ones. Instead, they should be collected then, if none are found before parsing headlines, it should default to ["TODO", "DONE"]

**Comment by transitracer** [comment]
I do think there's a more essential problem here though that needs to be addressed, actually, in that orga needs to take two passes so that it can handle in-buffer settings like with todo keywords. While it's reasonable to assume they'll be at the top of most files, that can't be assumed to be universally true since it isn't enforced in org mode

**Comment by xiaoxinghu** [comment]
This is fixed in v1.5.3. It can be separated from #28. Fell free to reopen, or create another one if any issues.
